docs/introduction/syncing-overview.md

```diff
@@ -29,33 +29,33 @@ syncs, listen for push notifications and upload events.
 
 1.  When the santad process starts up, it looks for a `SyncBaseURL` key/value in
     the config. If one exists it will `fork()` and `execve()` `santactl sync
-    —-daemon`. Before the new process calls `execve()`, all privileges are
-    dropped. All privileged actions are then restricted to the XPC interface
+    —-daemon`. All privileges are dropped before the new process calls `execve()`.
+    All privileged actions are then restricted to the XPC interface
     made available to santactl by santad. Since this santactl process is running
     as a daemon it too exports an XPC interface so santad can interact with the
     process efficiently and securely. To ensure syncing reliability santad will
     restart the santactl daemon if it is killed or crashes.
-2.  The santactl daemon process now schedules a full sync for 15 sec in the
+3.  The santactl daemon process now schedules a full sync for 15 sec in the
```

Comment:
This renumbering is incorrect - 2 is missing.

Reply:
fixed instead of it going 1-3-4-5-6 it is not 1-2-3-4-5